### DISCUSSION TITLE
Should Finder close windows opened by a Bunch?

### DISCUSSION THREAD

**Discussion by MatthewHeun** [ttscoff/bunch-docs#102] [Q&A]
Greetings:

Thanks, Brett, for Bunch. Just heard about it on Automators and am loving it!

One question about this simple bunch:
```
---
title: Finder test
shortcut: f
---

# Hide all other apps
@@

Finder
- ~/Desktop
```
When the `Finder test` bunch closes, should the Finder window containing `/Desktop` also close? 

I expect that closing the bunch *would* close the Finder window containing `/Desktop`. However, the `/Desktop` window remains open when the bunch closes, at least for me. 

Am I missing a configuration or a setting somewhere? Is this a bug? Is there a workaround to ensure that the Finder window containing `/Desktop` closes when the bunch closes?

Note this question is similar to #81 but different.

Thanks again!

**Comment by ttscoff** [comment]
Because most people have multiple Finder windows at any given time, and the simple act of opening a subfolder makes a window unrecognizable, there's not really an elegant way to clean up Finder windows when closing a Bunch.

You could write an applescript to do so for your own needs, though, and run it when closing:

!* my_script.scpt

- Brett
